react/pages/profile/ProfilePage/components/EmptyMessageOrComponent/fragments/emptyOrTips.js

```diff
@@ -7,10 +7,6 @@ export default gql`
       id
       created_at(format: "%Y-%m-%dT%H:%M:%S.%L%z")
       is_me
-      counts {
-        blocks
-        channels
-      }
```

Comment:
No, these look like they belong here?

Reply:
They no longer do.

Reply:
Well sorta, they would need to either be in a fragment for `ProfileViews` or rather just be handled within this component.

Reply:
It's unclear what they belong to here

Reply:
Ah, they exist on the fragment above. Shoot, guess this is more just cleaning up the logic here (rather than removing the need for a count).